docs/modules/ROOT/pages/accounts.adoc

```diff
@@ -4,48 +4,52 @@
 
 = Accounts
 
-Unlike Ethereum where accounts are directly derived from a private key, there's no native Externally Owned Account (EOA) concept on Starknet.
+Unlike Ethereum where accounts are derived from a private key, all Starknet accounts are contracts. This means there's no Externally Owned Account (EOA)
+concept on Starknet.
 
-Instead, the L2 features native account abstraction, and signature validation has to be done at the contract level. To relieve smart contract applications such as ERC20 tokens or exchanges from this responsibility, we make use of Account contracts to deal with transaction authentication.
+Instead, the network features native account abstraction and signature validation happens at the contract level.
 
-For a general overview of the account abstraction, see StarkWare's https://medium.com/starkware/starknet-alpha-0-10-0-923007290470[StarkNet Alpha 0.10].
-A more detailed discussion on the topic can be found in https://community.starknet.io/t/starknet-account-abstraction-model-part-1/781[StarkNet Account Abstraction Part 1].
+For a general overview of the account abstraction, see
+https://docs.starknet.io/documentation/architecture_and_concepts/Accounts/introduction/[Starknet's documentation].
+A more detailed discussion on the topic can be found in
+https://community.starknet.io/t/starknet-account-abstraction-model-part-1/781[Starknet Shaman's forum].
 
 TIP: For detailed information on the usage and implementation check the xref:/api/account.adoc[API Reference] section.
 
-== Table of Contents
-
-* <<basic_account,Basic Account>>
-** <<account_entrypoints,Account entrypoints>>
-** <<interface,Interface>>
-* <<standard_account_interface,Standard Account Interface>>
-** <<isrc6_interface,ISRC6 Interface>>
-** <<isrc5_interface,ISRC5 Interface>>
-* <<deploying_an_account,Deploying an Account>>
-
 == Basic Account
 
-As we mentioned, Accounts in Starknet are smart contracts, and so they can be deployed and interacted with like any other contract, and can be extended to implement any custom logic. However, the Account is a special type of contract that is used to validate and execute transactions, and there are a set of entrypoints that must be implemented for enabling these features.
-
+As we mentioned, accounts in Starknet are smart contracts, and so they can be deployed and interacted
```

Comment:
following up on our [_basic account_ discussion](https://github.com/OpenZeppelin/cairo-contracts/pull/709#discussion_r1311114516), we should link to SNIP-6 as early as in this section